fern/pages/fern-docs/content/customize-api-ref.mdx

```diff
@@ -0,0 +1,190 @@
+---
+title: Customize API Reference layout
+description: Customize your API Reference's naming, ordering, and structure.  
+---
+
+When you include an API (either the Fern or an OpenAPI definition) in your `docs.yml` file, you can customize how the endpoints and sub-packages are displayed in the sidebar navigation. By default, the reference will generate a navigation hierarchy based on the structure of the API spec, but several customizations can be configured.
+
+## Ordering the API Reference
+
+### Flattening sections
+To remove the API Reference title and display the section contents, set `flattened` to `true`.
+
+```yaml title="docs.yml"
+navigation: 
+  - api: API Reference
+    flattened: true
+```
+
+<Frame>
+<img src="flattened.png" alt="Flattened API Reference" />
+</Frame>
+
+### Alphabetizing endpoints and sections
+To sort all sections and endpoints alphabetically, unless explicitly ordered in `layout`, set `alphabetized` to `true`.
+
+
+```yaml title="docs.yml"
+navigation: 
+  - api: API Reference
+    alphabetized: true
+```
+
+### Ordering top-level sections
+The `layout` option allows you to specify the order of subpackages, sections, and endpoints at the top level of your API Reference.
+
+<Note title="Section names">
+If you are using an OpenAPI Defintiion, sections are created based on the `tags` property. If you are using a Fern Definition, sections are created based on the `service` file names.
+</Note>
+
+<Tabs>
+<Tab title="Ordering sections">
+```yaml title="docs.yml"
+navigation: 
+  - api: API Reference
+    layout: 
+      - user
+      - store
+      - plant
+```
+</Tab>
+
+<Tab title="Ordering sections and endpoints">
+```yaml title="docs.yml"
+navigation: 
+  - api: API Reference
+    layout: 
+      - user.create
+      - user
+      - store
+      - plant
+```
+
+<Frame>
+<img src="ordered.png" alt="Ordered API Reference" />
+</Frame>
+</Tab>
+</Tabs>
+
+### Ordering section contents
+Adding a `:` after the section name allows you to specify the order of its nested subpackages and endpoints. 
+
+<Note title="Referencing Endpoints">
```

Comment:
Instead of a note, can we use tabs?

`docs.yml for OpenAPI`
`docs.yml for Fern Definition`

Reply:
I'd want to show both to users (right now, we just show the Fern Def example)

Reply:
Maybe best then to default to the "METHOD path" naming, since that is most obvious to both types.

Reply:
I'd hesitate to separate into tabs, since they are not technically exclusive to each

Reply:
Sat with this - agree with you here